platform/web/js/engine/config.js

```diff
@@ -183,34 +183,49 @@ const InternalConfig = function (initConfig) { // eslint-disable-line no-unused-
 		 */
 		onProgress: null,
 		/**
-		 * A callback function for handling the standard output stream. This method should usually only be used in debug pages.
+		 * A callback function for handling prints. This method should usually only be used in debug pages.
 		 *
 		 * By default, ``console.log()`` is used.
 		 *
 		 * @callback EngineConfig.onPrint
-		 * @param {...*} [var_args] A variadic number of arguments to be printed.
+		 * @param {string} text The text to be printed.
```

Comment:
The previous docstring said that this callback takes a variadic number of arguments to be printed. However, this callback has only ever took one argument to it because `Module.print` only takes one argument.

Variadic arguments are handled differently in JS than in C++ so any implementation of this function utilizing the variadic arguments through the spread operator `...` would still work as intended even after this change.